tests/lean/PPRoundtrip.lean.expected.out

```diff
@@ -40,7 +40,7 @@ Type → Type → Type
   [inst :
     ToString Nat],
   x = x
-∀ (x : Nat), x = x
+∀ x, x = x
```

Comment:
Previously, `pp.binderTypes false` was getting overridden by `pp.analyze`, which would still annotate all `pi`s to need binder types. Now, `pp.analyze` does not add annotations `pi` binders.